src/components/JoinUs/Learning.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { Padding, Margin } from 'styled-components-spacing'
+import styled from 'styled-components'
+import { Row, CompensatedRow, CompensatedCol } from '../grid'
+import Image from '../Common/Image'
+import { H2, Paragraph } from '../Typography'
+
+import {
+  Section,
+  FirstColumn,
+  SecondColum,
+  Separator,
+  List,
+  Hr,
+  TitleAndBody
+} from './elements'
+
+const InsightAnchor = styled.a`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+`
+
+const Learning = ({ data: { title, text, featuredInsights, list } }) => (
+  <Section greyBg>
+    <Row>
+      <FirstColumn>
+        <H2>{title}</H2>
+      </FirstColumn>
+      <SecondColum width={[1, 1, 1, 1, 6 / 12, 5 / 12]}>
```

Comment:
Based on [this line](https://github.com/yldio/yld.io/pull/80/files#diff-68a74b60a6cd09266fc4ff3341c7a014R19) in the definition of SecondColumn, should this be achieved by adding the `fullWidth` prop?

Reply:
you're right

Reply:
I mean, not setting the prop